backend/fms_core/template_importer/importers/extraction.py

```diff
@@ -0,0 +1,73 @@
+from fms_core.models import Protocol, Process, SampleKind
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.extraction import ExtractionRowHandler
+
+
+class ExtractionImporter(GenericImporter):
+    SHEETS_INFO = [
+        {'name': 'ExtractionTemplate', 'header_row_nb': 6},
+    ]
+
+
+    def __init__(self):
+        super().__init__()
+        # Preload objects accessible to the whole template (not only by row)
+        self.preload_data_from_template()
+
+
+    def preload_data_from_template(self):
+        self.preloaded_data = {'process': None, 'sample_kinds': {}}
+
+        self.preloaded_data['process'] = Process.objects.create(protocol=Protocol.objects.get(name="Extraction"),
```

Comment:
This would not qualify as loading ... I do not oppose bunching up preloading and global creation together but we might want to rename the function... initialize_data_for_template() ?